packages.json

```diff
@@ -4,20 +4,16 @@
             "all": [
                 "alsa-firmware",
                 "android-udev-rules",
-                "apr",
-                "apr-util",
                 "distrobox",
                 "ffmpeg",
                 "ffmpeg-libs",
                 "ffmpegthumbnailer",
-                "fzf",
                 "google-noto-sans-balinese-fonts",
                 "google-noto-sans-cjk-fonts",
                 "google-noto-sans-javanese-fonts",
                 "google-noto-sans-sundanese-fonts",
                 "grub2-tools-extra",
                 "heif-pixbuf-loader",
-                "htop",
```

Comment:
This was added over a year ago ( #82 ) as an opinionated, lightweight helpful utility. The only concern I've ever seen about this is that it comes with a desktop file which shows up in DE app lists and some people seem to find that offensive.

I'm not inclined to remove it.

Reply:
> The only concern I've ever seen about this is that it comes with a desktop file which shows up in DE app lists and some people seem to find that offensive.

This one can be easy solved with:
echo 'Hidden=true' >> /usr/share/applications/htop.desktop

Reply:
Yep, I am fine with hiding the desktop icon if the group wishes to do that.

Reply:
> Yep, I am fine with hiding the desktop icon if the group wishes to do that.

I would also do that for `nvtop` too, if majority agrees.

Reply:
I'm -1 for mangling .desktop files in main - I agree it's annoying, which is why we do it downstream. 😄

Reply:
Also agreed, though, I'd suggest that be a distinct PR.

Reply:
@bsherman ultimately this PR is about giving `main` a well-defined scope, which it currently lacks. Without which, it is not clear at all what belongs here, upstream, or downstream.

If these packages are universally useful, they should be added in fedora comps.

Reply:
@qoijjj I just addressed this in top-level comment.